Automattic/wp-calypso
# Subscribers Page: Added export as csv to Subscribers Page

Closes https://github.com/Automattic/wp-calypso/issues/78364

## Proposed Changes

This PR adds the "export to CSV" feature to the Subscribers Page. This will export a CSV file with the emails of the email subscribers.

**Note:** Only the email subscribers will appear in the CSV, so you can find more people in the list on the Subscribers Page than in the CSV file.

## Testing Instructions

1. Apply this PR and start the application.
2. Go to `http://calypso.localhost:3000/subscribers/{the domain of your site}`.
3. Click on the ellipsis menu in the top right corner of the screen and then on the `Download email subscribers as CSV` link.
<img width="1107" alt="Screenshot 2023-06-20 at 18 20 07 (1)" src="https://github.com/Automattic/wp-calypso/assets/3832570/77d17d34-626a-4fca-8aca-eb7ecc4ba235">
4. It should download a file with the name like `[domain-of-the-site]-[id-of-the-site]-email`. If you open it, you will see the list of emails of the email subscribers on the Subscribers Page.


## Pre-merge Checklist
